README.md

```
@@ -63,7 +66,14 @@ state-of-the-art optimizations to perform inference efficiently on NVIDIA GPUs.<
 * [05/16] Pushing Latency Boundaries: Optimizing DeepSeek-R1 Performance on NVIDIA B200 GPUs
 ✨ [➡️ link](https://nvidia.github.io/TensorRT-LLM/blogs/tech_blog/blog1_Pushing_Latency_Boundaries_Optimizing_DeepSeek-R1_Performance_on_NVIDIA_B200_GPUs.html)
 
+</details>
+
 ## Latest News
+* [2026/03/05] Suffix Automaton, contributed by Baseten, adds further acceleration to MTP, Eagle, and PARD [➡️ link](https://www.baseten.co/blog/boosting-mtp-acceptance-rates-in-baseten-speculation-engine/)
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)